### DISCUSSION TITLE
Windowed UI

### DISCUSSION THREAD

**Discussion by L2750558108** [godotengine/godot-proposals#6024] [Editor]
With windowed editors like Unity and Visual Studio.Windows can be docked on the dock or hovered over, rather than being repositioned only by the ellipsis button.Also includes options fixed at the top such as 2D, 3D scene editor, and options fixed at the bottom such as output, debugger, etc. They will be able to detach from the bottom and top docks and float above the screen or dock on the side docks.

![image](https://user-images.githubusercontent.com/119792950/210047286-e6d690f4-30d6-42ed-b91d-0d6d29af9bb0.png)

**Comment by RedMser** [comment]
See also https://github.com/godotengine/godot-proposals/issues/4565
